activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Consistently raise an `ArgumentError` if the cache key is blank.
```

Comment:
Maybe mention that it has to do with ActiveSupport::Cache.

Reply:
Updated!